src/twisted/test/test_plugin.py

```diff
@@ -261,6 +261,8 @@ def test_detectFilesChanged(self):
                 True)
 
     test_detectFilesChanged = _withCacheness(test_detectFilesChanged)
+    if _PYPY:
+        test_detectFilesChanged.skip = "Does not work on PYPY"
```

Comment:
This test works fine for me under PyPy 5.8.0 under 2.7 and 3.5 modes. there is something weird/different with PyPy2 5.8.0 buildbot environment that messes this up and have not been able to get to bottom of it yet.